cmd/format.go

```diff
@@ -1,22 +1,43 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/formatter"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+const failFastFlagName = "fail-fast"
+
 func setupFormatCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "format",
 		Short: "Format the package",
 		Long:  "Use format command to format the package files.",
 		RunE:  formatCommandAction,
 	}
+	cmd.Flags().BoolP(failFastFlagName, "f", false, "fail if any file requires formatting")
 	return cmd
 }
 
 func formatCommandAction(cmd *cobra.Command, args []string) error {
-	fmt.Println("format is not implemented yet.")
+	packageRoot, ok, err := packages.FindPackageRoot()
+	if err != nil {
+		return errors.Wrap(err, "locating package root failed")
+	}
+	if !ok {
+		return errors.New("package root not found")
+	}
+
+	ff, err := cmd.Flags().GetBool(failFastFlagName)
+	if err != nil {
+		return errors.Wrapf(err, "flag not found (flag: %s)", failFastFlagName)
```

Comment:
Not for this PR but as we start parsing flags in various command implementations we should probably standardize on the error message when the flag is not found. Maybe create a common function to generate the error or something like that in `internal/cmd/util.go` (related: https://github.com/elastic/elastic-package/pull/53/files#r470018706).

Reply:
Good idea, Shaunak. I prepared such method, also applied in couple of places.